packages/icons/metadata.yml

```diff
@@ -833,6 +833,14 @@ icons:
         categories:
           - name: Organization
             subcategory: File
+      - name: folder--off
```

Comment:
@chrisconnors-ibm do we want to add metadata in this pass or hold off till later?

Reply:
they have to have a category/subcategory; have to have a friendly name. In theory synonyms could wait, but Strongly Recommend they be included.

Reply:
they have to have a category/subcategory; have to have a friendly name. In theory synonyms could wait, but Strongly Recommend they be included.

Reply:
@chrisconnors-ibm do we want this included in the next release? If so, who should I track down to help with contributing the appropriate meta info?